src/test/java/com/wikia/webdriver/elements/mercury/components/Head.java

```diff
@@ -0,0 +1,25 @@
+package com.wikia.webdriver.elements.mercury.components;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class Head {
+
+  private final WebDriver driver;
+
+  @FindBy(css = "head title")
+  private WebElement documentTitle;
+
+  public Head(WebDriver driver) {
+    this.driver = driver;
+    PageFactory.initElements(driver, this);
+  }
+
+  public String getDocumentTitle() {
+    JavascriptExecutor js = (JavascriptExecutor) driver;
```

Comment:
WikiBasePageObject is the way to go for now, it's getting smaller and smaller every time, and we will clean that up finally. There (in `WBPO`) are few things we would like to keep - driver providing mechanisms, `PageFactory` init and basic `wait` and `jsActions` objects. Extending it will give you few bonuses:
1. You don't have to carry about drivers and object initialisation
2. You don't have to pass any drivers from a test body (components are aware which driver is the active one)
3. You have your wait and jsActions objects up front (two most common used ones)
All in all - less noise in components, and cleaner test (finally - no mythical `driver` object in test body :) )

 I know  we are missing updated guidelines, and that is a good point. We will get there ;)